en/python_introduction/README.md

```diff
@@ -633,7 +633,7 @@ And on Windows Powershell, it will be like this:
 <!--endsec-->
 
 
-If you get stuck, just ask for help.
+If you get stuck, ask for help.
```

Comment:
Asking for help is actually a very difficult thing for some people to do. This is why we need to remove the "just", and why the Coach's Handbook suggests not using language like that.

Reply:
I'd be OK with adding in a "don't worry" in there. But I still think "just" implies "it's easy to ask for help", which is why the Coach's handbook suggests not using language like that.

Reply:
Hmm ... yeah. Maybe "don't worry" is better than "just".

Reply:
I have noticed students hesitating to ask questions because they don't want to disturb the coach while I'm there bored if they don't ask questions. How about something like:

> If you get stuck, ask for help. That's exactly what the coaches are here for!